sonar-project.properties

```diff
@@ -0,0 +1,5 @@
+sonar.projectKey=pantos-io_client-cli
```

Comment:
The key should probably be `pantos-io_client-library`.

Reply:
Changed. Thanks